mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
+# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
```

Comment:
not sure what this does, just rubber stamping for now

Reply:
this essentially tells vscode what schema it should expect the file to match. the yaml language server in vscode knows to parse this directive even when it's commented out. but when it's not commented out, it gets passed as a config param into mkdocs too which gets confused and outputs warnings - which is why it needs to be commented

Reply:
Can we add a comment above it specifying this?

Reply:
Done.